src/config.js

```diff
@@ -10,13 +10,14 @@ import Fbw from '../plugins/lime-plugin-fbw';
 
 // REGISTER PLUGINS
 export const plugins = [
-	Rx,
-	Align,
-	Locate,
-	Metrics,
-	Notes,
-	Meta,
-	//GroundRouting,
-	Fbw,
-	Admin
-];
\ No newline at end of file
+  Rx,
+  Align,
+  Locate,
+  Metrics,
+  Notes,
+  Meta,
+  // GroundRouting,
+  Fbw,
+  Admin,
+  Pirania
```

Comment:
don't change format please, as it breaks change tracking (like in this file Pirania is the only thing added, but it changes the whole file. thanks!